### PR TITLE
Special guidance for ECS contributions during OTel donation

### PR DESCRIPTION
**Pull request by norrietaylor**
## Overview
Elastic has [announced](https://www.elastic.co/blog/ecs-elastic-common-schema-otel-opentelemetry-faq) that it will donate ECS to the OpenTelemetry Semantic Conventions. In this announcement, we shared our intention to achieve convergence of ECS and OTel Semantic Conventions into a single standard maintained by OpenTelemetry.

This documentation provides the process and guidelines for contributing to ECS during the OTel donation process.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -22,12 +23,57 @@ ECS is an open source project and we love to receive contributions from our comm
...
-1. The [RFC process](./rfcs/README.md) is used for significant additions or breaking changes to the schema itself.
-2. For bug fixes or incremental, non-controversial additions to ECS, changes can be made directly to the ECS project and submitted as pull request.
+## ECS Donation to OpenTelemetry
+In April 2023, OpenTelemetry and Elastic made an [important joint announcement](https://opentelemetry.io/blog/2023/ecs-otel-semconv-convergence/). In this announcement Elastic
+shared its intention to achieve convergence of ECS and OTel Semantic Conventions into a single standard maintained
+by OpenTelemetry.
+
+The stated plan has been to keep ECS in a frozen state during the transition. However, it is also apparent that these
+things take time. It takes time for the OTel community to adopt donated fields, and it will take time for development
+teams to build OTel native constructs in the Elastic stack. In the meantime, our internal and external ECS users need
```
Comment (ebeahan) [inline CONTRIBUTING.md:36]:
```suggestion
teams to build OTel native constructs in the Elastic stack. In the meantime, our ECS users need
```

nit: I think we can refer to just `ECS users` from the project's standpoint.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -22,12 +23,57 @@ ECS is an open source project and we love to receive contributions from our comm
...
+allows us to avoid breaking changes.
+
+### How to contribute during OTel donation of ECS
+
+Bug fixes or minor field addition changes can be made directly to the ECS project and submitted as pull requests.
+
+Significant changes that add new use cases, top-level fieldsets, or could be considered controversial are
+considered material. The general rule for contributing new material changes to schemas during the transition period is
+
+- First, merge a pull request to OTel with new fields, namespaces or schemas
```
Comment (ebeahan) [inline CONTRIBUTING.md:50]: Should we add a pointer here to which OTel project(s) to submit a PR?

**Review by ebeahan — APPROVED** [review]
LGTM. A couple of nits I didn't consider in my first read through.

File: CONTRIBUTING.md (hunk 1 above)
Comment (norrietaylor) [reply CONTRIBUTING.md:36]: Good catch

**Pull request merged**